carsus/io/output/base.py

```diff
@@ -10,15 +10,14 @@
 import astropy.constants as const
 from scipy import interpolate
 from datetime import datetime
-from carsus.util import (convert_wavelength_air2vacuum,
+from carsus.util import (convert_atomic_number2symbol,
+                         convert_wavelength_air2vacuum,
                          serialize_pandas_object,
                          hash_pandas_object)
 from carsus.model import MEDIUM_VACUUM, MEDIUM_AIR
 
-# TODO: pass GFALL_AIR_THRESHOLD as parameter
-# [nm] wavelengths above this value are given in air
-GFALL_AIR_THRESHOLD = 200
-
+# Wavelengths above this value are given in air
+GFALL_AIR_THRESHOLD = 2000*u.AA
```

Comment:
```suggestion
GFALL_AIR_THRESHOLD = 2000 * u.AA
```
PEP8!!!

Reply:
and we are in angstrom - not sure if we should switch back and forth?

Reply:
As I said before, readers return values in `nm` and the `output` module in `AA`.

Switching back and forth seems complicated, but it's necessary for comparing against legacy code. Remember this part of the code does not have tests (should be an integration test?) so the only way to check we are not breaking anything is to make a small database (i.e. `H-Be`) and compare dataframe by dataframe.